android/src/main/java/ch/milosz/reactnative/RNZoomUsInMeetingServiceListener.java

```diff
@@ -0,0 +1,297 @@
+    }
+
+    @Override
+    public void onMeetingUserUpdated(long l) {
+
+    }
+
+    @Override
+    public void onMeetingHostChanged(long l) {
+
+    }
+
+    @Override
+    public void onMeetingCoHostChanged(long l) {
+
+    }
+
+    @Override
+    public void onActiveVideoUserChanged(long l) {
+        notifyEvent("meeting.user.video.active", extractUserInfoFromUserId(l));
+    }
+
+    @Override
+    public void onActiveSpeakerVideoUserChanged(long l) {
+        notifyEvent("meeting.user.video.speaker", extractUserInfoFromUserId(l));
+    }
+
+    @Override
+    public void onSpotlightVideoChanged(boolean b) {
+
+    }
+
+
+    @Override
+    public void onUserNetworkQualityChanged(long l) {
+
+    }
+
+    @Override
+    public void onMicrophoneStatusError(InMeetingAudioController.MobileRTCMicrophoneError mobileRTCMicrophoneError) {
+
+    }
+
+    @Override
+    public void onUserAudioStatusChanged(long l) {
+        InMeetingUserInfo userInfo = inMeetingService.getUserInfoById(l);
+        WritableMap map = Arguments.createMap();
+        map.putString("name", userInfo.getUserName());
+        map.putString("userId", "" + userInfo.getUserId());
+        map.putBoolean("muted", userInfo.getAudioStatus().isMuted());
+        notifyEvent("meeting.user.audio.status", map);
+    }
+
+    @Override
+    public void onHostAskUnMute(long l) {
+
+    }
+
+    @Override
+    public void onHostAskStartVideo(long l) {
+
+    }
+
+    @Override
+    public void onUserAudioTypeChanged(long l) {
+
+    }
+
+    @Override
+    public void onMyAudioSourceTypeChanged(int i) {
+
+    }
+
+    @Override
+    public void onLowOrRaiseHandStatusChanged(long l, boolean b) {
+
+    }
+
+    @Override
+    public void onMeetingSecureKeyNotification(byte[] bytes) {
+
+    }
+
+    @Override
+    public void onChatMessageReceived(InMeetingChatMessage inMeetingChatMessage) {
+
+    }
+
+    @Override
+    public void onSilentModeChanged(boolean b) {
+
+    }
+
+    @Override
+    public void onFreeMeetingReminder(boolean b, boolean b1, boolean b2) {
+
+    }
+
+    @Override
+    public void onMeetingActiveVideo(long l) {
+    }
+
+    @Override
+    public void onSinkAttendeeChatPriviledgeChanged(int i) {
+
+    }
+
+    @Override
+    public void onSinkAllowAttendeeChatNotification(int i) {
+
+    }
+
+    @Override
+    public void onUserNameChanged(long l, String s) {
+
+    }
+
+//    @Override
+//    public void onUserVideoStatusChanged(long l, VideoStatus videoStatus) {
+//        WritableMap params = extractUserInfoFromUserId(l);
+//        String status;
+//        switch (videoStatus) {
+//            case Video_ON:
+//                status = "on";
+//                break;
+//            case Video_OFF:
+//                status = "off";
+//                break;
+//            case Video_Mute_ByHost:
+//                status = "muteByHost";
+//                break;
+//            default:
+//                status = "undefined";
```

Comment:
1. In lib `videoStatus` is ignored
2. In lib this event is triggered only for own user